docs/getting-started/wallets.md

```diff
@@ -1,289 +1,102 @@
 ---
-title: "Create Wallet"
+title: "Wallets and keys in Bittensor"
 ---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
 
-# Create Wallet
+# Wallets and keys in Bittensor
 
-This section describes steps to create a Bittensor wallet, regenerate keys, and encrypt keys. If you are new to Bittensor wallets, see [Working with Keys](../subnets/working-with-keys.md) for an explanation of Bittensor wallet keys.
+In Bittensor (like other cryptocurrency applications), a *wallet* is a tool for managing the cryptographic key-pairs required to prove your identity, sign transactions, and access your currency.
 
-## Ways of creating wallet
+This page introduces the core concepts of Bittensor cryptography. Procedures for handling wallets and keys are described in: [Working with keys](../working-with-keys.md)
 
-You can create a Bittensor wallet either for basic uses like securely storing your TAO and receiving and sending them or for advanced uses like creating a subnet and participating as a subnet miner or a subnet validator:
+## Coldkey and hotkey
 
-- **For basic use**: Create an external wallet account by using the [Chrome Extension for Bittensor Wallet](https://chromewebstore.google.com/detail/bittensor-wallet/bdgmdoedahdcjmpmifafdhnffjinddgc?pli=1). An external wallet account created in this way will allow you to use TAO **without installing Bittensor**. If your activities are limited to sending or receiving TAO, then this is a recommended option.
-- **For subnet participation**: Create a local wallet account using `btcli` command line tool on your computer. This requires that you install Bittensor on your machine. If you are interested in either creating a subnet or participating as a subnet miner or a subnet validator, then you must use this option.
+A Bittensor wallet consists of a **coldkey** and a **hotkey**. The coldkey and hotkey are used for different operations in the Bittensor ecosystem. These two keys are logically connected via the Bittensor API.
```

Comment:
Most users won't need a hotkey. They might just hold TAO and/or delegate it to a Validator(s)'s hotkey. 
Might be good to caveat it. All miners and validators will have both, but users that are neither will only ever use a coldkey.

Miners may even have multiple hotkeys per coldkey.